protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -65,34 +83,49 @@ protocol SimpleFS {
   record Progress {}
 
   record ListResult {
-    array<Item> items;
+    array<Dirent> Dirents;
     Progress progress;
   }
-
   
   record FileContent {
     bytes data;         // Zero bytes when read is complete
     Progress progress;  // Amount of data copied so far during this and previous reads
   }                     // (TBD: should this be percentage?)
+  
+  enum PathType {
+    LOCAL_0,
+    KBFS_1
+  }  
+
+  @typedef("string")
+  record LocalPath {}
+
+  @typedef("string")
+  record KBFSPath {}
+  
+  variant Path switch (PathType PathType) {
+    case LOCAL: LocalPath;
+    case KBFS: KBFSPath;
+  }
 
   record OpDescription {
     OpID opID;
-    string command;     // list, listRecursive, read, write, copy, move, remove
-    string src;         // source or path
-    string dest;        // destination if copy or move
+    string command;   // list, listRecursive, read, write, copy, move, remove
+    Path src;         // source or path
+    Path dest;        // destination if copy or move
```

Comment:
I agree with @chrisnojima, this should be a `variant` and behave different per-op.  And the command should be an `enum`.